src/js/manifest2015.js

```diff
@@ -1,275 +0,0 @@
-'use strict';
```

Comment:
Removed this file as we don't need it anymore.  Unlikely we will ever use these files anymore so may as well remove.